package.json

```diff
@@ -27,5 +30,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^13.13.1",
```

Comment:
Nitpick.

You could upgrade Cypress to `13.13.2`.

Just run `npm i cypress@latest -D`.

Then, add the changes and commit.